frontend/SideDish/SideDish/View/HeaderView.swift

```diff
@@ -0,0 +1,17 @@
+
+import UIKit
+
+class HeaderView: UICollectionReusableView {
+
+    @IBOutlet weak var headerButton: UIButton!
```

Comment:
IBOutlet에서 접근제어자 설정할 수 있습니다.

```suggestion
    @IBOutlet private weak var headerButton: UIButton!
```

Reply:
앗 감사합니다! 내부에서만 사용하는 것이라서 private로 하는 것이 더 좋겠네요!